### PR TITLE
Id 123 new feature

### PR DESCRIPTION
**Pull request by rotorrest**
Nice new feature

### CODE REVIEW

**Review by Spacecowboy423 — COMMENTED** [review]
I have reviewed these changes. Looks good :)

File: new_feature.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+print("new feature")
+print("new feature 1")
```
Comment (Spacecowboy423) [inline new_feature.py]: This should be in ALL CAPS!

File: new_feature.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+print("new feature")
```
Comment (Baba101722) [inline new_feature.py]: Great work today.

File: new_feature.py (hunk 1 above)
Comment (rotorrest) [reply new_feature.py]: Hey @Spacecowboy423, changes made. now its with caps =)

**Review by Spacecowboy423 — APPROVED** [review]
B-E-A-UTIFUL!

**Review by Baba101722 — APPROVED** [review]

**Pull request merged**